test/util/tools.ts

```diff
@@ -22,14 +22,18 @@ async function getBlockNumber(): Promise<number> {
   return parseInt(blockNumber);
 }
 
-async function skippedRoundsIncrease(n: number, price: number): Promise<number> {
-  const maxIncreaseRate = 1036;
-  const multiplier = 1024;
+async function skippedRoundsIncrease(
+  n: number,
+  price: number,
+  priceBaseString: number,
+  maxIncreaseRate: number
+): Promise<number> {
+  const base = priceBaseString;
```

Comment:
just called it `base` in the argument definitions, this assignment does not do anything than just renaming the variable